src/snapred/backend/dao/StateConfig.py

```diff
@@ -15,7 +16,14 @@ class StateConfig(BaseModel):
     rawVanadiumCorrectionFileName: str  # Needs to be removed when Normalization changes go in
     vanadiumFilePath: str  # Needs to be removed when Normalization changes go in
     focusGroups: List[FocusGroup]  # PixelGroupingParameters
-    stateId: str  # generated.
+    stateId: ObjectSHA  # generated.
```

Comment:
Is this guaranteed to be the same as the `_generateStateId` method in `LocalDataService`?

Reply:
There's only _one_ `_generateStateId` method, and it's in `LocalDataService`.  That should be the only way a `stateId` is ever filled in.  **Note that `_generateStateId` still generates exactly the same hex digest as it did before.**  The only thing our changes in this PR do is _standardize_ the hex digest to a 16-charater lowercase hex string.